eslint.config.js

```diff
@@ -11,9 +11,6 @@ module.exports = [
     {
         rules: {
             camelcase: ["error", { properties: "never" }],
-
-            // Remove after https://github.com/eslint/eslint/pull/17900 is included in new release
-            "jsdoc/no-multi-asterisks": ["error", { allowWhitespace: true }]
```

Comment:
This is how this rule is now configured in eslint-config-eslint, so no need to override.

https://github.com/eslint/eslint/blob/362099c580992b2602316fc417ce3e595b96f28c/packages/eslint-config-eslint/base.js#L224